api-samples/storage/stylizr/popup.js

```diff
@@ -0,0 +1,36 @@
+// Store CSS data in the "local" storage area.
+const storage = chrome.storage.local;
+
+const message = document.querySelector('#message');
+
+// Check if there is CSS specified.
+(async function () {
+  const items = await storage.get('css');
+  if (items.css) {
+    const [currentTab] = await chrome.tabs.query({
+      active: true,
+      currentWindow: true
+    });
+    try {
+      await chrome.scripting.insertCSS({
+        css: items.css,
+        target: {
+          tabId: currentTab.id
+        }
+      });
+      message.innerText = 'Injected style!';
+    } catch (e) {
+      message.innerText = 'Not allowed to inject CSS into special page.';
```

Comment:
Could we:

- Update the message to make it a bit less presumptuous. Maybe "Injection failed. Are you on a special page?"
- Log the error to the console as well